app/controllers/state_file/questions/df_return_sample.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
this is a default json sample for direct file return sample xml. (i.e. old sample in the flows controller)